src/Dialect/ONNX/ONNXOps.cpp

```diff
@@ -51,8 +51,9 @@ static mlir::ONNXConstantOp getONNXConstantOp(Value value) {
 //===----------------------------------------------------------------------===//
 // Get reduction type
 //===----------------------------------------------------------------------===//
-RankedTensorType getReductionOutputType(
-    RankedTensorType operandTy, Optional<ArrayAttr> axesAttrs, APInt keepdims) {
+RankedTensorType getReductionOutputType(RankedTensorType operandTy,
+                                        Optional<ArrayAttr> axesAttrs,
```

Comment:
I don't believe that you are following the formatting defined under onyx-mlir/.clang-format